### ISSUE TITLE
refactor the image credits styles in old blog posts

### ISSUE THREAD

**Issue by alicetragedy** [rails-girls-summer-of-code/summer-of-code#289]
In one of the [last contributions to the blog](https://github.com/rails-girls-summer-of-code/summer-of-code/pull/288/files#diff-a96549d4a2a4c3e9ce19652830b23434R421), the inline styling we use for image credits/descriptions was moved [to its own class in a css file](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/assets/stylesheets/blog.scss#L42). Since we'll be sticking to that convention in the future (to have as little code in the single post files as possible) I'd like to fix all of the past posts as well to use that same convention.

**Comment by frefell** [comment]
is this still relevant? If yes, could you provide a little more context or which posts this would apply to? I'm just beginning but would like to help :)

**Comment by v-thakkar** [comment]
Hi @frefell . Thanks for your interest. This is still relevant. The issue here is we have old posts which are not using image-credits style. You can check 2015 and 2014 posts. And for example you can check 2017 posts which are using image-credit style. I think you can start with picking the file -> 2015-10-30-cocoagems-get-strange-at-strangeloop.md.

**Comment by frefell** [comment]
Thanks for clarifying. I will start with 2015 then :)

**Comment by mkalininait** [comment]
Fixed blog posts of 2017 (https://github.com/rails-girls-summer-of-code/summer-of-code/commit/273a9c56e95debf2f5d14e7e266d37e5f732012f)

**Comment by alicetragedy** [comment]
thank you @mkalininait ! \o/

**Comment by MasonHolland** [comment]
@alicetragedy are the only pieces remaining on this the 2016 posts?

**Comment by alicetragedy** [comment]
@MasonHolland yes, I think so! I'd probably have to double-check to make sure that nothing else is missing, but please don't let that stop you from contributing :) If you're interested in cleaning up the 2016, go ahead — in the meanwhile I'll take a look at what we have so far and if there's any older posts we might have missed.

**Comment by alicetragedy** [comment]
@MasonHolland just wondering — are you interested in working on this issue? Is there anything I can do to help?

**Comment by wiseemily88** [comment]
@alicetragedy- Hi! I’m a first-time contributor and was hoping to help out with this issue & I noticed in the comments that @MasonHolland is working on it. I'd love to help, and wanted to check in on the current status for this issue. Is there currently a solution in progress?

**Comment by alicetragedy** [comment]
hej @wiseemily88 — thank you so much for your interest in helping out 💖I haven't heard anything back from @MasonHolland so my suggestion would be that you just go ahead and pick up this issue, if you want to work on it. 
I believe the 2016 posts are the only ones left; let me know if you need any guidelines on getting started or if you get stuck or need clarifications on the task — I'm always happy to help!
